packages/adapter-d1/README.md

```diff
@@ -27,7 +27,7 @@ datasource db {
 }
 ```
 
-Install Prisma Client, Prisma adapter for Cloudflare D1, Prisma CLI, Cloudflare's workers types and Wrangler CLI packages:
+Install the Prisma Client, the Prisma adapter for Cloudflare D1, the Prisma CLI, the TypeScript types for Cloudflare Workers, and Wrangler CLI packages:
```

Comment:
@nikolasburk can speak about this, we do not use "the" for Prisma Client and co is what I remember.

Reply:
yup, "Prisma Client" is a proper noun and doesn't get prefixed with an article.